### ISSUE TITLE
thoughness error?

### ISSUE THREAD

**Issue by MercuryTempest** [SpaceXVI/TAIGA#8]
Jauxum, Ovium, Karmesine can be broken with vanilla stone pickaxe.

**Comment by SpaceXVI** [comment]
Yes it is! I think it's with other ores too but currently I can't do much about it until post-July due to my studying...

I am sorry for any inconveniences.

**Comment by MercuryTempest** [comment]
> Yes it is! I think it's with other ores too but currently I can't do much about it until post-July due to my studying...
> 
> I am sorry for any inconveniences.

¡Luck on your studying! 
¿is there any chance to suggest something? 
I don't want you to make a brand new alloy out of literally nowhere but it feels very lineal to have only 1 tier 5, 6 and 7 that can be made with it's last tier items : vibranium for example, vibranium pickaxe is lvl 7 tier but it's made with vibranium which only can be destroyed with tier 7, so you enter in a loop where you only can exit by making firstly a Solarium pickaxe.

If I could suggest a new metal it would've a freeze like metal (found in snowy taiga, snowy mountains, etc?) I feel like it would fit for a meterial of a tier 5 alloy. Tier 5 asks for materials from Nether-End-Overworld and obsidian, ¿what about changuing obsidian for this new material? the new material wouldn't work for anything really, as some materials doesn't do anything for themselves (if I don't remember wrong, seismum and fractum doesn't work for anything), but alloying it would make a material (like purifing the material by adding it to other materials) with modifier "freeze cold" (like sticky modifier in tinker's traveller's set) it would stuck. Enemies with slowness, slowness 1 if freeze cold I, 2 if II, etc. (centered in combat/self defense metal).

And another idea, for a tier 7 material obtainable with tier 6 gear (without making a solarium pickaxe first) would've a duranite and/or valyrium replacement material in the tier 7 alloy recipe. This new material would've a material that spawns near end cities, it would've a purple material (endum?), for it's alloying you would need Nucleum + Duranite/Valyrium + Endum = Nightum, leuksum? (leuksna in latin is moon). 

Im so sorry if this bothers you, or is not firstly made on actual Taiga 1 but I really feel is "weird" to have a lineal progression on tiers on a mod that gives you the chance to have a lot of materials and alloys that are obtainable in different ways having or not the last tier (talking about tinkers), for example,you would start with tier 1-2 with wood, flint or stone, you then you can choose from slimewood or copper for iron and you start your first smeltery and with alloys you can start making whatever you want, you can go with bronze, nahuatl for diamond tier or practically any alloy that is diamond tier but doesnt include debris or cobalt, not having to get extrictly "Flint > Wood > Stone > Copper > Iron > diamond alloys > netherite alloys > diamond alloys but made with netherite alloys"

**Comment by SpaceXVI** [comment]
I highly appreciate your suggestions! I like the idea of giving more options for the players to work their way up, I didn't think much about it! I will make sure to try implementing some of these ideas when I am done with my last year of school.

Have a good day!
